scripts/generate_llms.sh

```
@@ -0,0 +1,706 @@
+exclude_pred=""
+for d in $EXCLUDE_DIRS; do
+  exclude_pred="$exclude_pred -path '*/$d' -prune -o"
+done
+
+# Temp workspace (no associative arrays in macOS Bash 3.2)
+SECTION_TMP_DIR="$(mktemp -d)"
+trap '[[ -d "$SECTION_TMP_DIR" ]] && rm -rf "$SECTION_TMP_DIR"' EXIT
+
+# Existing LLMS map for incremental LLM usage: url \t desc
+EXISTING_MAP="${SECTION_TMP_DIR}/existing_map.tsv"
+> "$EXISTING_MAP"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)